src/components/Navbar/Navbar.tsx

```diff
@@ -3,28 +3,24 @@ import { Menu } from '../Menu/Menu';
 export const Navbar = () => {
   return (
     <header>
-      <nav className="fixed top-0 h-20 w-[100%] mx-auto overflow-hidden flex items-center justify-between bg-bubble-gum">
+      <nav className="fixed top-0 h-16 w-[100%] z-20 mx-auto overflow-hidden flex items-center justify-between bg-background-navbar">
```

Comment:
Try always to use the base classes like w-full  w-1/2, and w-5/6... they are there for spacing consistency. Do not change this now.